cookie_booths/views.py

```diff
@@ -324,14 +326,50 @@ def is_block_reserved_by_user(request, block_id):
     return
 
 
+def get_week_start_end_from_date(date):
+    start_date = date - timedelta(days=date.weekday())
+    end_date = start_date + timedelta(days=6)
+
+    return start_date, end_date
+
+
+def get_num_tickets_remaining(troop, date):
```

Comment:
Basically copied from users/models.py - moved here to remove circular dependency so cookie_booths/models.py has access to Troop/User for email purposes.

Reply:
We may want to consider creating a module for these items that don't really have a good home. Nothing that needs to change now.